k8s/external-secrets/telegraf.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kubernetes-client.io/v1
+kind: ExternalSecret
+metadata:
+  name: telegraf-influx
+  namespace: telegraf-stage
+spec:
+  backendType: secretsManager
+  dataFrom:
+    - /stage/telegraf
```

Comment:
1 smaller notes - do we want a comment or similar here to help inform the expected secret json structure, since we don't get that from this aggregated secret? Just an ease of comprehension question for understanding what gets passed here to `envFromSecret`. Have another question about this structure, but going to check myself first.

Reply:
Sorry, was this off base?

Reply:
Ah, I see the changes. Thanks for adding that docs in line.